### DISCUSSION TITLE
Can't install discord.py python 3.8.6

### DISCUSSION THREAD

**Discussion by woqwoq** [Rapptz/discord.py#5950] [Q&A]
Microsoft Windows [Version 10.0.19041.508]
(c) Корпорация Майкрософт (Microsoft Corporation), 2020. Все права защищены.

C:\Users\Влад>pip install discord
Collecting discord
  Downloading discord-1.0.1-py3-none-any.whl (1.1 kB)
Collecting discord.py>=1.0.1
  Using cached discord.py-1.5.0-py3-none-any.whl (699 kB)
Collecting aiohttp<3.7.0,>=3.6.0
  Using cached aiohttp-3.6.3.tar.gz (1.1 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... error
    ERROR: Command errored out with exit status 1:
     command: 'C:\Users\Влад\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\python.exe' 'C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.8_3.8.1776.0_x64__qbz5n2kfra8p0\lib\site-packages\pip\_vendor\pep517\_in_process.py' prepare_metadata_for_build_wheel 'C:\Users\46AD~1\AppData\Local\Temp\tmpkldqf00s'
         cwd: C:\Users\Влад\AppData\Local\Temp\pip-install-qyh3d0ee\aiohttp
    Complete output (24 lines):
    running dist_info
    creating C:\Users\Влад\AppData\Local\Temp\pip-modern-metadata-_74h3gl3\aiohttp.egg-info
    writing C:\Users\Влад\AppData\Local\Temp\pip-modern-metadata-_74h3gl3\aiohttp.egg-info\PKG-INFO
    writing dependency_links to C:\Users\Влад\AppData\Local\Temp\pip-modern-metadata-_74h3gl3\aiohttp.egg-info\dependency_links.txt
    writing requirements to C:\Users\Влад\AppData\Local\Temp\pip-modern-metadata-_74h3gl3\aiohttp.egg-info\requires.txt
    writing top-level names to C:\Users\Влад\AppData\Local\Temp\pip-modern-metadata-_74h3gl3\aiohttp.egg-info\top_level.txt
    writing manifest file 'C:\Users\Влад\AppData\Local\Temp\pip-modern-metadata-_74h3gl3\aiohttp.egg-info\SOURCES.txt'
    reading manifest file 'C:\Users\Влад\AppData\Local\Temp\pip-modern-metadata-_74h3gl3\aiohttp.egg-info\SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    Error in sitecustomize; set PYTHONVERBOSE for traceback:
    SyntaxError: (unicode error) 'utf-8' codec can't decode byte 0xc2 in position 0: invalid continuation byte (sitecustomize.py, line 21)
    warning: no files found matching 'aiohttp' anywhere in distribution
    warning: no previously-included files matching '*.pyc' found anywhere in distribution
    warning: no previously-included files matching '*.pyd' found anywhere in distribution
    warning: no previously-included files matching '*.so' found anywhere in distribution
    warning: no previously-included files matching '*.lib' found anywhere in distribution
    warning: no previously-included files matching '*.dll' found anywhere in distribution
    warning: no previously-included files matching '*.a' found anywhere in distribution
    warning: no previously-included files matching '*.obj' found anywhere in distribution
    warning: no previously-included files found matching 'aiohttp\*.html'
    no previously-included directories found matching 'docs\_build'
    writing manifest file 'C:\Users\Влад\AppData\Local\Temp\pip-modern-metadata-_74h3gl3\aiohttp.egg-info\SOURCES.txt'
    creating 'C:\Users\Влад\AppData\Local\Temp\pip-modern-metadata-_74h3gl3\aiohttp.dist-info'
    error: invalid command 'bdist_wheel'
    ----------------------------------------
ERROR: Command errored out with exit status 1: 'C:\Users\Влад\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\python.exe' 'C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.8_3.8.1776.0_x64__qbz5n2kfra8p0\lib\site-packages\pip\_vendor\pep517\_in_process.py' prepare_metadata_for_build_wheel 'C:\Users\46AD~1\AppData\Local\Temp\tmpkldqf00s' Check the logs for full command output.
WARNING: You are using pip version 20.2.1; however, version 20.2.4 is available.
You should consider upgrading via the 'C:\Users\Влад\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\python.exe -m pip install --upgrade pip' command.

I have tried to reinstall python but it not helped.

**Comment by nihaals** [comment]
You should avoid using the Microsoft Store version of Python and instead install it from the [standard installer](https://www.python.org/downloads/release/python-386/)

Also, you are installing `discord` instead of `discord.py`. It is recommended to do `pip install discord.py` instead

**Reply by crypticknot** [reply]
Still not working :(((

**Comment by Harmon758** [comment]
Make sure you have setuptools installed and that it's up to date, e.g. `pip install --upgrade setuptools`.
If it still errors, try installing the latest version of wheel, e.g. `pip install --upgrade wheel`.

For code block usage, see https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks.

**Reply by Ech-Dev** [reply]
**thank you very much I had the same problem and this worked**
